tornjak-frontend/src/components/types.ts

```diff
@@ -0,0 +1,99 @@
+// agents
+
+export interface SPIFFEIDType {
```

Comment:
wouldn’t it be redundant to specify type? Unless this is idiomatic in typescript, which I suspect not. @florentinvintila  please validate.